shared/store/configure-store.js

```diff
@@ -1,16 +1,16 @@
 // @flow
-import {Iterable} from 'immutable'
-import configureStoreNative from './configure-store.platform'
 import createLogger from 'redux-logger'
 import createSagaMiddleware from 'redux-saga'
+import storeEnhancer from './enhancer.platform'
+import mainSaga from './configure-sagas'
 import rootReducer from '../reducers'
 import thunkMiddleware from 'redux-thunk'
+import {Iterable} from 'immutable'
```

Comment:
:sheep: Reordering the imports in the same diff like this makes figuring out what changed a lot slower. Could you please separate out formatting changes like this into a separate commit?